vitest.config.mts

```diff
@@ -0,0 +1,14 @@
+import react from "@vitejs/plugin-react";
+import tsconfigPaths from "vite-tsconfig-paths";
+import { defineConfig } from "vitest/config";
+
+// eslint-disable-next-line import/no-default-export
+export default defineConfig({
+  plugins: [tsconfigPaths(), react()],
+  test: {
+    exclude: ["node_modules", "src/e2e/**"],
```

Comment:
potrzebny jest exclude na node_modules?

Reply:
jak odpalałem testy to mi też zaczynało testować moduły w `mode_modules` więc dałem na niego exclude, sprawdzę czy wciąż tak jest